XenModel/Actions/OvfActions/Import.cs

```diff
@@ -121,158 +70,96 @@ public static void Process(IXenConnection connection, EnvelopeType ovfObj, strin
 
         	XenRef<VM_appliance> applRef = null;
 			if (applianceName != null)

```

Comment:
Nit: please replace the tab indentation here with spaces.

Reply:
I'll include it in the next PR.